joern-cli/frontends/php2cpg/build.sbt

```diff
@@ -6,6 +6,9 @@ name := "php2cpg"
 scalaVersion       := "2.13.8"
 crossScalaVersions := Seq("2.13.8", "3.2.2")
 
+lazy val phpParserVersion = settingKey[String]("php-parser version")
```

Comment:
do you ever plan to change this setting other than in this place, e.g. via `sbt phpParserVersion := "some-other-version" compile`? My guess would be 'no', in which case this doesn't need to be an sbt setting but rather a simple value. I.e. simply `val phpParserVersion = "4.15.4-charfix"` and drop the `= settingKey` line above. 
Same goes for `phpParserDlUrl`.

Reply:
That's a good point! I do think the answer is no, so I'll go simplify that.